### PR TITLE
🌐 Add Russian translation of the main page

### PR DESCRIPTION
**Pull request by CharNikita**
@tiangolo Hello! Started translating documentation into Russian. 
Issue #1362

### CODE REVIEW

**Comment by ArcLightSlavik** [comment]
There's already a `index.md` translation in https://github.com/tiangolo/fastapi/pull/1363
But as ycd/kludex noted in https://github.com/tiangolo/fastapi/issues/1362 the creator of the PR seems to be gone, so if u want u can take over that PR in this one. This includes other files separate from `index.md`.
On a side note I will probably need to do the same with Ukrainian 🙂

**Comment by CharNikita** [comment]
> There's already a `index.md` translation in #1363
> But as ycd/kludex noted in #1362 the creator of the PR seems to be gone, so if u want u can take over that PR in this one. This includes other files separate from `index.md`.
> On a side note I will probably need to do the same with Ukrainian 🙂

**Comment by tiangolo** [comment]
Awesome!

I'll leave this PR and close the previous one at https://github.com/tiangolo/fastapi/pull/1363 as the original author's account for that other PR is no longer available in GitHub.

Now this just needs a couple of reviews. :eyes: :mag: :heavy_check_mark:

File: docs/ru/docs/index.md [hunk 1]
```diff
@@ -22,29 +22,28 @@
...
 
-* **Fast**: Very high performance, on par with **NodeJS** and **Go** (thanks to Starlette and Pydantic). [One of the fastest Python frameworks available](#performance).
+* **Эффективность**: Отличная производительность наравне с **NodeJS** и **Go** (благодаря Starlette и Pydantic). [Один из самых быстрых Python фреймворков](#performance).
+* **Скорость разработки**: Увеличьте свою скорость разработки на 200–300%. *
+* **Меньше багов**: На 40% меньше багов при разработке. *
+* **Авто дополнение**: Отличная поддержка IDE – тратьте меньше времени на отладку.
+* **Простота**: Низкий порог вхождения. Простая и понятная документация.
+* **Лаконичность**: Минимальное дублирование кода. Большое количество готовых решений.
+* **Надёжность**: Получите production-ready код с генерацией интерактивной документации.
+* **Специфицирован**: Основан, и полностью совместим, на стандартах API: <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (так же известный как Swagger) и <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>.
```
Comment (solomein-sv) [inline docs/ru/docs/index.md:42]:
```suggestion
* **Соответствие стандартам**: Основан на открытых стандартах API: <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (ранее известный как Swagger) и <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>, и полностью совместим с ними.
```

File: docs/ru/docs/index.md [hunk 2]
```diff
@@ -98,24 +97,24 @@ The key features are:
 
 ---
 
-## **Typer**, the FastAPI of CLIs
+## **Typer** – FastAPI для CLI приложений
 
 <a href="https://typer.tiangolo.com" target="_blank"><img src="https://typer.tiangolo.com/img/logo-margin/logo-margin-vector.svg" style="width: 20%;"></a>
 
-If you are building a <abbr title="Command Line Interface">CLI</abbr> app to be used in the terminal instead of a web API, check out <a href="https://typer.tiangolo.com/" class="external-link" target="_blank">**Typer**</a>.
+Если вы разрабатываете <abbr title="Command Line Interface">CLI</abbr>-приложение обратите внимание <a href="https://typer.tiangolo.com/" class="external-link" target="_blank">**Typer**</a>.
```
Comment (solomein-sv) [inline docs/ru/docs/index.md:104]:
```suggestion
Если вы разрабатываете <abbr title="Command Line Interface">CLI</abbr>-приложение, обратите внимание на <a href="https://typer.tiangolo.com/" class="external-link" target="_blank">**Typer**</a>.
```

File: docs/ru/docs/index.md [hunk 3]
```diff
@@ -127,7 +126,7 @@ $ pip install fastapi
 
 </div>
 
-You will also need an ASGI server, for production such as <a href="https://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a> or <a href="https://gitlab.com/pgjones/hypercorn" class="external-link" target="_blank">Hypercorn</a>.
+Вам также понадобится ASGI сервер например <a href="https://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a> или <a href="https://gitlab.com/pgjones/hypercorn" class="external-link" target="_blank">Hypercorn</a>.
```
Comment (solomein-sv) [inline docs/ru/docs/index.md:129]:
```suggestion
Вам также понадобится ASGI сервер, например, <a href="https://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a> или <a href="https://gitlab.com/pgjones/hypercorn" class="external-link" target="_blank">Hypercorn</a>.
```

File: docs/ru/docs/index.md [hunk 4]
```diff
@@ -211,54 +210,54 @@ INFO:     Application startup complete.
...
 
-You will see the JSON response as:
+Вы увидите JSON-ответ:
 
 ```JSON
 {"item_id": 5, "q": "somequery"}
 ```
 
-You already created an API that:
+Мы создали API который:
```
Comment (solomein-sv) [inline docs/ru/docs/index.md:233]:
```suggestion
Мы создали API, который:
```

File: docs/ru/docs/index.md [hunk 5]
```diff
@@ -211,54 +210,54 @@ INFO:     Application startup complete.
...
 
-* Receives HTTP requests in the _paths_ `/` and `/items/{item_id}`.
-* Both _paths_ take `GET` <em>operations</em> (also known as HTTP _methods_).
-* The _path_ `/items/{item_id}` has a _path parameter_ `item_id` that should be an `int`.
-* The _path_ `/items/{item_id}` has an optional `str` _query parameter_ `q`.
+* Принимать HTTP запросы по _маршрутам_ `/` и `/items/{item_id}`.
```
Comment (solomein-sv) [inline docs/ru/docs/index.md:235]:
```suggestion
* Принимает HTTP запросы по _маршрутам_ `/` и `/items/{item_id}`.
```

File: docs/ru/docs/index.md [hunk 6]
```diff
@@ -211,54 +210,54 @@ INFO:     Application startup complete.
...
+* Оба _маршрута_ умеют обрабатывать `GET`-<em>операцию</em> ( HTTP _метод_).
```
Comment (solomein-sv) [inline docs/ru/docs/index.md:236]:
```suggestion
* Умеет обрабатывать `GET`-<em>операции</em> ( HTTP _метод_) для обоих _маршрутов_ .
```

File: docs/ru/docs/index.md [hunk 7]
```diff
@@ -211,54 +210,54 @@ INFO:     Application startup complete.
...
+*  _Маршрут_ `/items/{item_id}` имеет _параметр_ `item_id` типа `int`.
```
Comment (solomein-sv) [inline docs/ru/docs/index.md:237]:
```suggestion
* Для  _маршрута_ `/items/{item_id}` обрабатывает обязательный _параметр_ `item_id` типа `int`.
```

File: docs/ru/docs/index.md [hunk 8]
```diff
@@ -211,54 +210,54 @@ INFO:     Application startup complete.
...
+*  _Маршрут_ `/items/{item_id}` имеет необязательный _параметр_ `q` типа `str`.
```
Comment (solomein-sv) [inline docs/ru/docs/index.md:238]:
```suggestion
* Для_маршрута_ `/items/{item_id}` обрабатывает необязательный _параметр запроса_ `q` типа `str`.
```

File: docs/ru/docs/index.md [hunk 9]
```diff
@@ -211,54 +210,54 @@ INFO:     Application startup complete.
...
-You will see the alternative automatic documentation (provided by <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>):
+Это документация предоставлена <a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>:
 
 ![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
 
-## Example upgrade
+## Улучшаем пример
 
-Now modify the file `main.py` to receive a body from a `PUT` request.
+Изменим код `main.py`чтобы получить тело `PUT` запроса.
```
Comment (solomein-sv) [inline docs/ru/docs/index.md:258]:
```suggestion
Изменим код `main.py`, чтобы получить тело `PUT` запроса.
```

**Review by solomein-sv — APPROVED** [review]

File: docs/ru/docs/index.md [hunk 10]
```diff
@@ -22,29 +22,28 @@
...
-The key features are:
+Ключевые особенности:
 
-* **Fast**: Very high performance, on par with **NodeJS** and **Go** (thanks to Starlette and Pydantic). [One of the fastest Python frameworks available](#performance).
+* **Эффективность**: Отличная производительность наравне с **NodeJS** и **Go** (благодаря Starlette и Pydantic). [Один из самых быстрых Python фреймворков](#performance).
+* **Скорость разработки**: Увеличьте свою скорость разработки на 200–300%. *
+* **Меньше багов**: На 40% меньше багов при разработке. *
+* **Авто дополнение**: Отличная поддержка IDE – тратьте меньше времени на отладку.
+* **Простота**: Низкий порог вхождения. Простая и понятная документация.
+* **Лаконичность**: Минимальное дублирование кода. Большое количество готовых решений.
```
Comment (Ae-Mc) [inline docs/ru/docs/index.md:40]:
```suggestion
* **Лаконичность**: Минимальное дублирование кода. Большое количество готовых решений. Меньше багов.
```

**Pull request closed without merging**